components/advanced-search.tsx

```
@@ -0,0 +1,275 @@
+                }
+              />
+            </div>
+            <div className="space-y-2">
+              <Label>Max Commute Time</Label>
+              <Select
+                value={searchCriteria.commute?.maxTime?.toString()}
+                onValueChange={(value) =>
+                  setSearchCriteria({
+                    ...searchCriteria,
+                    commute: {
+                      ...searchCriteria.commute!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)